numalogic/transforms/_stateless.py

```diff
@@ -156,3 +156,45 @@ def transform(self, X: npt.NDArray[float], **__) -> npt.NDArray[float]:
 
     def inverse_transform(self, X: npt.NDArray[float]) -> npt.NDArray[float]:
         return X.reshape(-1, self.n_features)
+
+
+class FlattenVectorWithPadding(StatelessTransformer):
+    """A stateless transformer that flattens some of the columns and does padding on the rest.
```

Comment:
Can we have __slots__ defined here. We also might have to move 182 and 183 inside transform.

Reply:
Added